citeproc/model.py

```diff
@@ -756,7 +756,10 @@ def _term(self, item):
         if plural:
             text = term.multiple
         else:
-            text = term.single
+            try:
+                text = term.single
+            except Exception as e:
+                text = None
```

Comment:
We should [limit which exceptions are caught](https://www.quantifiedcode.com/knowledge-base/correctness/Avoid%20untyped%20exception%20handlers/3JwOg9ad). AttributeError in this case, I believe.

But in this case it's probably better to check whether `term` is equal to `None`.